components/style/values/specified/text.rs

```diff
@@ -514,3 +515,18 @@ impl ToComputedValue for TextAlign {
         TextAlign::Keyword(*computed)
     }
 }
+
+/// A specified value for the `-moz-tab-size` property.
+pub type MozTabSize = GenericMozTabSize<NonNegativeNumber, NonNegativeLength>;
+
+impl Parse for MozTabSize {
+    fn parse<'i, 't>(
+        context: &ParserContext,
+        input: &mut Parser<'i, 't>,
+    ) -> Result<Self, ParseError<'i>> {
+        if let Ok(number) = input.try(|i| NonNegativeNumber::parse(context, i)) {
```

Comment:
Maybe worth still pointing out that it's important that `NonNegativeNumber` is parsed before `NonNegativeLength`?

Reply:
Done.